client/src/domain/Community/Community.js

```diff
@@ -0,0 +1,89 @@
+import "./Community.css";
+import { Link } from "react-router-dom";
+
+import CommunityFallbackProfilePic from "../../assets/happy-students.jpg";
+
+import { FaRegEdit } from "react-icons/fa";
+import { FaTrashAlt } from "react-icons/fa";
+import axios from "axios";
+
+import CommunitiesContext from "../../contexts/CommunitiesContext";
+import { useContext } from "react";
+
+import { useHistory } from "react-router";
+
+// TODO: Get rid of that conditional wrapper again - shoudl be possible to just set an onLick-handler on the section-tag itself and basrta kanzla!
```

Comment:
If you have a TODO left, then why do you want to merge this into production? ;-)
A pull request should be only having working and production ready code.

Reply:
removed